src/aws-cpp-sdk-core/include/aws/core/client/AWSClient.h

```diff
@@ -356,6 +358,7 @@ namespace Aws
             Aws::String m_serviceName = "AWSBaseClient";
             Aws::Client::RequestCompressionConfig m_requestCompressionConfig;
             Aws::Vector<std::shared_ptr<smithy::interceptor::Interceptor>> m_interceptors;
+            Aws::Http::HeaderValueCollection m_featureHeaders;
```

Comment:
I guess smithy client needs to be updated as well.

Reply:
Ok, I will extend this to smithy client